src/Sylius/Bundle/ApiBundle/Command/Cart/ChangeItemQuantityInCart.php

```diff
@@ -25,19 +25,11 @@ class ChangeItemQuantityInCart implements OrderTokenValueAwareInterface, Subreso
     /** @var string|null */
     public $orderItemId;
 
-    /**
-     * @psalm-immutable
-     *
-     * @var int
-     */
-    public $quantity;
-
-    public function __construct(int $quantity)
+    public function __construct(public ?int $quantity)
     {
-        $this->quantity = $quantity;
     }
 
-    public static function createFromData(string $tokenValue, string $orderItemId, int $quantity): self
+    public static function createFromData(string $tokenValue, string $orderItemId, ?int $quantity): self
```

Comment:
If I understand correctly, we are changing the constructor because of validation, it is probably ok, but I have some doubts about that, however, here we still shouldn't be able to pass a null value
```suggestion
    public static function createFromData(string $tokenValue, string $orderItemId, int $quantity): self
```